plugins/chanbackup.c

```diff
@@ -760,6 +760,128 @@ static struct command_result *json_restorefrompeer(struct command *cmd,
 				     	    NULL);
 }
 
+/* Returns True if the backup data is correct and
+ * returns false with the reason in `fail` if it's incorrect.
+ * Instead of using `decrypt_scb()` we're using this because
+ * it shuts down the node in case of any discrepancy and that
+ * is not what we want to happen when someone runs `verifybackup`. */
+static bool verify_bkp(struct command *cmd,
+					   u8 *filedata,
+					   char **fail) {
```

Comment:
The indentation here look a little bit strange, the u8 should be in the same column of the struct

Reply:
Hmm, my editor's setting is all over the place, actually this is a different machine. Fixing in a bit, thanks for pointing out!